cmd/collect-statse/config/config.go

```diff
@@ -29,9 +30,10 @@ var Loaded *Config
 
 func Load(tsdbChan chan<- *tsdb.Point) {
 	var (
-		filePath  = flag.String("f", "/etc/collect-statse/config", "configuration file")
-		debugMode = flag.Bool("v", false, "verbose mode")
-		testMode  = flag.Bool("t", false, "configuration test")
+		filePath    = flag.String("f", "/etc/collect-statse/config", "configuration file")
+		debugMode   = flag.Bool("v", false, "verbose mode")
+		testMode    = flag.Bool("t", false, "configuration test")
+		VersionMode = flag.Bool("version", false, "output version and exit")
```

Comment:
s/V/v/ for consistency with surrounding code.

I realize there isn't one convention followed by all programs in this repo but addressing that belongs to a separate diff.